modules/main-rules/pages/transportation.adoc

```diff
@@ -2,12 +2,42 @@
 
 On some unit data sheets you'll see that the unit has one of two special abilities that relate to transportation: {transport-x} or {hitch-x}.
 We explain these and other special abilities in more detail elsewhere; but transportation is a vital aspect of the Movement phase so we'll discuss it here too.
+// IJW editing note - the second sentence isn't really true, this page is where  the detail is!
 
 When we speak of these transportation abilities, we use the terms 'embark/embarked' and 'disembark/disembarked' to describe actions and states that relate to the units that you transport.
 
 Both the {transport-x} and {hitch-x} abilities indicate units that can carry -- transport -- other units across the battlefield.
 But they work in slightly different ways.
 
+== Placement of units
+To embark a unit, it must be completely within 5 cm of the transportation unit.
```

Comment:
Is this section meant to cover both Transport (X) and Hitch (X), as it appears from the context?

If so then it seems to imply that Hitch (X) units no longer have to make contact with the units they will carry — only get close enough so that those units are fully within 5 cm. Is this right? I mean, movement is sacrificed for Transport (X) so it balances out, but that's not the case for Hitch (X) — so doesn't this change imply a significant boost to Hitch (X)? Or am I missing something that balances it out?

If not, what **does** apply to Hitch (X) as far as evening out base size issues for embarking and disembarking are concerned?

Reply:
> Is this section meant to cover both Transport (X) and Hitch (X), as it appears from the context?

Yes. As per https://github.com/GameBrains/er-core/issues/11#issuecomment-869613462 the main purpose of this branch is to apply the same rules for embarking and disembarking (apart from the different costs) to Hitch and Transport, in a way that reduces the impact of differing base sizes.

> If so then it seems to imply that Hitch (X) units no longer have to make contact with the units they will carry — only get close enough so that those units are fully within 5 cm. Is this right? I mean, movement is sacrificed for Transport (X) so it balances out, but that's not the case for Hitch (X) — so doesn't this change imply a significant boost to Hitch (X)? Or am I missing something that balances it out?

Well...

_Some_ movement is sacrificed for Transport.
**All** movement is sacrificed for Hitch.

But that's not the issue - Hitch is almost always used to drop units off, and then pick them up from the same place during the next move of the Hitch unit. This means that embarking and disembarking need to be exact antonyms in game mechanics. So if units can be dropped off completely within 5 cm (to avoid base size issues), they need to be picked up from completely within 5 cm so that Hitch actually works.

In practical terms, when using the standard strip bases, you 'gain' exactly the same 1 cm per embark and disembark for Hitch as you gain for Transport, because the far end of the base is 5 cm away from the transportation unit, instead of 4 cm for placing them in base contact. Smaller bases 'gain' more, but again that's the purpose of all the 'completely within 5 cm' stuff.

Reply:
>*Some* movement is sacrificed for Transport. **All** movement is sacrificed for Hitch.

It doesn't seem to be quite that straightforward to me. To be precise:

**In E40k**:
* For Transport (X), both the troops and the transport units sacrifice some movement.
* For Hitch (X), only the troops sacrifice movement — the transport units only sacrifice movement in as much as they have to deviate from their optimum course in order to embark the troops.

**But, in this draft for ER**: 
* It seems to me that we're effectively saying that Hitch (X) units now make even less sacrifice from the optimum course, and troop units *potentially* gain a bit. That is, they won't gain in the standard 'deployed and moving across the board' case, but they will gain in some other situations that are more typical of **re**deployment.
* This is because the transport only has to get close enough to case the troop unit to be fully within 5 cm — which in many cases will mean they only have to get within 3 to 4 cm.

So, have I identified an **actual** problem here that demands a bit more thought, or is it balanced out, insignificant or irrelevant because of some other part of the equation that I haven't considered?

Reply:
> * It seems to me that we're effectively saying that Hitch (X) units now make even less sacrifice from the optimum course, and troop units _potentially_ gain a bit. That is, they won't gain in the standard 'deployed and moving across the board' case, but they will gain in some other situations that are more typical of **re**deployment.

Yes, in exactly the same way that Transport (X) units make slightly less sacrifice from their optimum course in those situations. Remember that the primary aim for this branch is rationalising Transport (X) and Hitch (X), with rules changes from E40k because the polls about this were overwhelmingly in favour of updating the transportation rules.

> * This is because the transport only has to get close enough to case the troop unit to be fully within 5 cm — which in many cases will mean they only have to get within 3 to 4 cm.

No, it won't. 4 cm would require the transported unit's base to be 1 cm diameter*, and 3 cm would require the base to be 2 cm diameter. You can't even get an old square base completely within 5 cm and have it be 3 cm from the transportation unit.

*It could be oval and bigger on the longer axis, but oval infantry bases are incredibly rare.

> So, have I identified an **actual** problem here that demands a bit more thought, or is it balanced out, insignificant or irrelevant because of some other part of the equation that I haven't considered?

Embarkation and disembarkation have to be mirror-image manoeuvres to prevent weird results where you can drop off a unit but then can't pick up the dropped off unit in a subsequent phase, even though nobody has moved since then.

This makes _both_ forms of transportation slightly more flexible, but doesn't give either of them an advantage over the other.

If you have time at some point, could you give these rules a try on the game table? The need for embarkation and disembarkation to be mirrored manoeuvres gets a lot clearer in action.

Reply:
>No, it won't. 4 cm would require the transported unit's base to be 1 cm diameter*, and 3 cm would require the base to be 2 cm diameter. You can't even get an old square base completely within 5 cm and have it be 3 cm from the transportation unit.

Okay, mostly true — outside of some arrangements with larger transports that have long rectangular bases — so we're talking something like ~2.5cm maximum in most cases when you account for the arc of measurement. (Now that I think of it, I tend to picture a simple arrangement with a perpendicular measurement in my head — but I should really picture an arc.)

But as long as both types of transport are getting the same overall gain, and the gain is slight enough in the overall scheme of things, then I guess it won't matter — which seems to be the crux of your points.

>If you have time at some point, could you give these rules a try on the game table?

Unfortunately, I'm unlikely to make time for that any time soon, unless my main Epic gaming friend pops up and we get an actual proper game in. In any case, I don't disagree that embarkation and disembarkation should mirror each other. I'm just querying other **potential** implications or disparities that occur to me as I edit the words.

In this case it seems that I **haven't** identified an **actual** issue that needs more thought — so, I'm happy to move on.